test/integration/common/StudioActions.kt

```diff
@@ -235,15 +231,22 @@ object Utils {
         }
     }
 
-    private fun fileNameToString(fileName: String): String {
-        return Files.readAllLines(Paths.get(fileName), StandardCharsets.UTF_8).filter { line -> !line.startsWith('#') }
-            .joinToString("")
+    private fun readQueryFileToString(queryFileName: String): String {
+        return Files.readAllLines(Paths.get(queryFileName), StandardCharsets.UTF_8)
+            .joinToString("\n")
+    }
+
+    object Errors {
+        private const val CONNECT_TYPEDB = "Failed to connect to TypeDB."
```

Comment:
I would be more explicit with these names. `Errors.DATA_WRITE`, in particular, is unclear; `Errors.DATA_WRITE_FAILED` would be clearer.

Now, obviously this means every error currently will end in `_FAILED`. But that's fine - that's incidental repetition, not systematic. Note how our `ErrorMessages` classes in TypeDB, Studio and Client Java look - the message names all concisely describe the error, while the message bodies describe it in a user-friendly format.

Reply:
Understood. Done. I've raised an issue in typedb:
- https://github.com/vaticle/typedb/issues/6652